.gitignore

```diff
@@ -11,6 +11,4 @@ Saves/*
 error_log_*
 .vscode
 updates/
-Thumbs.db
```

Comment:
Why was this removed? I don't think the thumb files should be included in the repo.

Reply:
They have to be, otherwise the "jorge" command is broken.

Reply:
who cares lol

Reply:
What does that command even do?

Reply:
Bluscream, Try and learn XD

Reply:
If we really do have to keep an obscure inside joke like that in the game, there are better ways to do it than abusing thumbs files.

Reply:
Well, if you want you can just readd the Thumbs.db file in the gitignore @Regalis11 as I personally don't care about that too much.